### PR TITLE
Next v8 - Filters

### PR DESCRIPTION
**Pull request by bbazukun123**
- Up-version all filters to be v8.0.0 and use latest pixi v8 dependencies (including RCs)
- TODOs / Known Issues:
   - [x] WebGPU vec3 uniform not inferring the value from pixi Color object correctly. Once fix, these filters need to be verified
           - Color Replace
           - Glow
           - Simple Lightmap
           - Outline
           - + a few others that has single color options
           - _**UPDATE**: Reverted back to use Float32Array for vec3 colors as changing pixi color does not reflect the value_ changes correctly on the uniforms.
   - [x] Color Gradient: Restructure array uniforms for WebGPU
   - [x] Color Map: 'nearest' does nothing on WebGPU
   - [x] Glitch: Update WGSL
   - [x] ~~Displacement filter issue~~

### CODE REVIEW

File: bundle/package.json [hunk 1]
```diff
@@ -65,6 +65,7 @@
     "@pixi/filter-radial-blur": "8.0.0",
     "@pixi/filter-reflection": "8.0.0",
     "@pixi/filter-rgb-split": "8.0.0",
+    "@pixi/filter-shockwave": "8.0.0",
```
Comment (bigtimebuddy) [inline bundle/package.json:68]: Why do we need to remove shockwave?

Comment (bbazukun123) [reply bundle/package.json:68]: Hey Matt, we previously removed it due to it already existed in the older v8 rc, but has now moved back to this repo 👍

**Comment by bigtimebuddy** [comment]
In v8 we are moving Pixi to be a single package with sub package exports. I wonder if we should do the same thing with filters. Would eliminate having to manage and publish multiple packages.

**Comment by bbazukun123** [comment]
> In v8 we are moving Pixi to be a single package with sub package exports. I wonder if we should do the same thing with filters. Would eliminate having to manage and publish multiple packages.

Thanks for the raise @bigtimebuddy ! @Zyie has noted and created a ticket for this: https://github.com/orgs/pixijs/projects/2/views/4?pane=issue&itemId=49264154

**Pull request merged**